README.md

```diff
@@ -84,8 +84,6 @@ ARTIFACTORY_USERNAME=admin
 ARTIFACTORY_ACCESS_TOKEN=<your_access_token>
 TF_ACC=true
 ```
-`ARTIFACTORY_USERNAME` is not used in authentication, but used in several tests, related to replication functionality. 
```

Comment:
This is an unrelated change. Revert it please

Reply:
removed form the PR